src/category_theory/limits/shapes/binary_products.lean

```diff
@@ -701,7 +701,6 @@ end
 section prod_functor
 variables {C} [has_binary_products C]
 
--- FIXME deterministic timeout with `-T50000`
```

Comment:
I believe this is fixed; I mentioned it here: https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/-T50000.20challenge. I'm pretty sure it got fixed by #4376, though I'm not certain.